test/cypress/integration/printLogsSuccess.spec.js

```diff
@@ -0,0 +1,7 @@
+describe('Print Logs Success.', () => {
+
+  it('Print Logs Success', () => {
```

Comment:
Please write a spec that tests that, in case we have two tests in a spec file first passing and the last not passing, and we have configured print to file 'onFail' that only the last test gets output.